pkg/scheduler/objects/queue.go

```diff
@@ -444,12 +445,25 @@ func (sq *Queue) setResources(guaranteedResource, maxResource *resources.Resourc
 		log.Log(log.SchedQueue).Debug("guaranteed resources setting ignored: cannot set zero guaranteed resources",
 			zap.String("queue", sq.QueuePath))
 	}
+
+	// For shim side we already make sure the max apps > 0 and valid parsing
+	maxAppsInt, err := strconv.ParseInt(maxApps, 10, 64)
+	maxAppsUint := uint64(maxAppsInt)
+	if err == nil {
+		log.Log(log.SchedQueue).Debug("setting max running apps",
+			zap.String("queue", sq.QueuePath),
+			zap.Uint64("current", sq.maxRunningApps),
+			zap.Uint64("new", maxAppsUint))
+	}
+	sq.maxRunningApps = maxAppsUint
+
+	//nolint:godox //TODO, we need max apps metrics and event change?
```

Comment:
Remove TODO, I think if you don't change the method then the linter won't complain either.

Reply:
Thanks @pbacsko for review, create a jira to do this:
https://issues.apache.org/jira/browse/YUNIKORN-2798